opsdroid/parsers/mention.py

```diff
@@ -0,0 +1,37 @@
+"""A helper function for parsing and executing mention skills."""
+
+import logging
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def parse_mention(opsdroid, skills, message):
+    """Parse a message if the bot is mentioned."""
+    matched_skills = []
+    mentions = {}
+
+    for connector in opsdroid.connectors:
+        if connector.config["name"] == "matrix":
+            mentions.update({"matrix": connector.config["mxid"]})
+
+    for skill in skills:
+        for matcher in skill.matchers:
+            if "mention" in matcher:
+                if "matrix" in mentions:
+                    if "formatted_body" in message.raw_event["content"] and (
+                        '<a href="https://matrix.to/#/' + mentions["matrix"] + '">'
```

Comment:
Am I wrong to think that the difference between these two is that one uses double quotes after `href=` and the other uses a single quote?

A way to simplify this could be by using format or fstrings. This could be turn into 
`'<a href="https://matrix.to/#/{mentions}">'.format(mentions=mentions["matrix"])`.

Or... use regex to check if `message.raw_event["content"]["formatted_body"]` matches any of `https://matrix.to/#/ mentions["matrix"]`.

Either way it should work, I'd go with regex just to avoid the same check just with different quotes.

Reply:
You're right. Was a last-minute patch because I noticed the test uses single quotes. Think I'll go for a regex